package.json

```diff
@@ -46,7 +46,7 @@
     "@reduxjs/toolkit": "^1.9.2",
     "@swapr/core": "^0.3.19",
     "@swapr/periphery": "^0.3.22",
-    "@swapr/sdk": "1.8.2",
+    "@swapr/sdk": "SwaprHQ/swapr-sdk#feat/add-biswap-dex-as-routeable-platform",
```

Comment:
Is this right?

Reply:
I think it's contextually right. Until we deploy the new SDK version with all the changes, this PR will remain as a Draft.

This version change is part of this draft for testing purposes (in case someone pulls the branch and wants to test around). As soon as we have the new SDK version, I'll update this commit and move it out from Draft :D

Same goes for this draft PR: https://github.com/SwaprHQ/swapr-dapp/pull/1828

Reply:
Gotcha, as long as it doesn't make it to production... feels risky though. But there is a big yellow sign on the PR so ok.